src/apis/reminder.api.js

```diff
@@ -1,47 +1,58 @@
 module.exports = (() => {
   const express = require('express');
-  const { ReminderService, SessionService } = require('../services');
+  const {
+    ReminderService,
+    SessionService,
+    EventService,
+    ActivityService
+  } = require('../services');
 
   const router = express.Router();
 
   router.get('/', SessionService.authenticateSession, (req, res) => {
-    ReminderService.findAllReminders()
-      .then(reminders =>
-        reminders.map(reminder => {
-          const id = reminder._id;
-          const { name, startDate, endDate } = reminder;
-          let event = null;
-          if (reminder.event) {
-            event = {
-              id: reminder.event._id,
-              name: reminder.event.name,
-              date: reminder.event.date
-            };
-          }
-          return { id, name, startDate, endDate, event };
-        })
-      )
-      .then(reminders => res.send(reminders))
+    var all = [];
+    ActivityService.findAllActivities()
+      .then(activities => {
+        all = all.concat(activities);
+        return EventService.findAllEvents();
+      })
+      .then(events => {
+        all = all.concat(events);
+        res.send(events);
```

Comment:
also, do not just send everything that's returned from the service. Think of the api layer as the interface to expose the business logic. No extra logic should be added by this layer.